app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -162,6 +169,23 @@ class StatePlayerRecyclerViewAssembler private constructor(
 
   val isCorrectAnswer = ObservableField<Boolean>(false)
 
+  fun saveState(bundle: Bundle) {
+    bundle.putInt(KEY_TRACKED_WRONG_ANSWER_COUNT, hintHandler.trackedWrongAnswerCount)
+    bundle.putProto(KEY_PREVIOUS_HELP_INDEX, hintHandler.previousHelpIndex)
+    bundle.putInt(KEY_HINT_SEQUENCE_NUMBER, hintHandler.hintSequenceNumber)
+    bundle.putBoolean(KEY_IS_HINT_VISIBLE_IN_LATEST_STATE, hintHandler.isHintVisibleInLatestState)
```

Comment:
@BenHenning using proto in the bundle does help but how can I add all these variables in a single proto.
I've never worked on proto before

Reply:
Hey Farees. Protos are just data structures that can contain strings, ints, booleans, other proto messages, etc. I suggest looking at how other protos are defined and their usage to get an idea on how to create this new one.

Reply:
Done